deploy/sre-prometheus/README.md

```diff
@@ -0,0 +1,115 @@
+# Explanations
+
+## Burn Rates
+### Explanation
+Burn rates define the rate at which the SLO error budget is used.
+A burn rate of 1 means the complete error budget is used after the complete rolling window.
+### Details
+In the basic implementation a couple of questions need to be asked:
+* How fast to we need to be alerted?
+* How much budget can we burn worst case until we're alerted?
+* How fast do we want to recover?
+
+The combinaiton of the first and the last question hints that we always want to be implementing at least a short
+and a long window.
+
+We want to avoid flapping alerts therefore a good time period to chose for a condition to exist
+is 1h. We now combine that with the time to recover, which should in this case not exceed 5m
+This means that a condition has to exist for the last 1h **and** 5m.
```

Comment:
Yes but this please a bigger role when recovering from the alert. Because it takes significantly longer to reduce your error rate when looking at a 60m period as opposed to a 5m period

Reply:
I get that aspect. Now im confused about if this comment is explaining the initial time to alert or time to recover?

Reply:
"We now combine that with the time to recover"
Both :-)